pennylane/templates/layers.py

```diff
@@ -130,35 +130,70 @@ def StronglyEntanglingLayers(weights, wires, ranges=None, imprimitive=CNOT):
         :width: 60%
         :target: javascript:void(0);
 
-
     Args:
-        weights (array[float]): array of weights of shape ``(L, len(wires), 3)``
-        wires (Sequence[int]): sequence of qubit indices that the template acts on
+<<<<<<< HEAD
+        weights (array[float]): array of weights of shape ``(:math:`L`, :math:`M`, 3)``
+        wires (Sequence[int] or int): int or sequence of qubit indices that the template acts on
+        ranges (Sequence[int]): sequence determining the range hyperparameter for each subsequent layer; if None
+=======
+        weights (array[float]): weights of shape ``(:math:`L`, :math:`M`, 3)``
+        wires (Sequence[int] or int): qubit indices that the template acts on
 
     Keyword Args:
-        ranges (Sequence[int]): sequence determining the range hyperparameter for each subsequent layer
+        ranges (Sequence[int]): Sequence determining the range hyperparameter for each subsequent layer; if None
+>>>>>>> f06cc819ae808ffde1255d34dd716aceab7ff477
```

Comment:
merge conflict :fearful:

Reply:
Sorry! I fixed and searched for merge conflicts in path now...